### PR TITLE
.1 eth balance minimum requirement

### LINKED ISSUE #30: Fee Calculation: executeOrder()
## Description
Issues may arise due to the inability to collect a fee when executeOrder, invoked by ops/gelato, fails due to an insufficient eth balance in the margin account to cover the execution fee. 
Since this type of failure happens during the interaction between the third party and the system, it requires a more thought-out approach. 

## Solution 
For example, having an additional mechanism to update the margin account owner about execution issues related to their pending limit orders

### PR DESCRIPTION
**Pull request by JChiaramonte7**
fixes #30

### CODE REVIEW

File: contracts/MarginBase.sol [hunk 1]
```diff
@@ -612,6 +612,10 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
         onlyOwner
         returns (uint256)
     {
+        require(
+            address(this).balance >= 1 ether / 10,
+            "Min 0.1 ETH balance in account"
+        );
```
Comment (JChiaramonte7) [inline contracts/MarginBase.sol:618]: Let me know if you think this is better with a custom error

Comment (JaredBorders) [reply contracts/MarginBase.sol:618]: I think it would be good to have a custom error here

Comment (JaredBorders) [reply contracts/MarginBase.sol:618]: could also use `100 finney` or `1e17 wei` to avoid division which costs gas

Comment (JChiaramonte7) [reply contracts/MarginBase.sol:618]: I believe there's compiler optimization step that evaluate these expressions and turns them into constant values, but will make the update for the custom error

**Comment by JChiaramonte7** [comment]
also resolves #60

solidity by default reverts if an invalid enum val is passed in. added a test to verify this

**Comment by JChiaramonte7** [comment]
@JaredBorders ready for re-review

**Review by JaredBorders — APPROVED** [review]
Looks good!

**Pull request merged**